server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/Worker.scala

```diff
@@ -194,16 +194,16 @@ private[deploy] class Worker(
       response.expiredShuffleKeys.asScala.foreach(shuffleKey => workerInfo.releaseSlots(shuffleKey))
       cleanTaskQueue.put(response.expiredShuffleKeys)
     } else {
-      logError("Worker not registered in master, clean all shuffle data and register again.")
+      logError("Worker not registered in master, clean expired shuffle data and register again.")
       // Clean all shuffle related metadata and data
-      cleanup(shuffleKeys)
+      cleanup(response.expiredShuffleKeys)
```

Comment:
shuffleKeys defined above can be deleted

Reply:
> shuffleKeys defined above can be deleted

Used in `Heatbeat` message